### DISCUSSION TITLE
Drop-down profile options for "Suggestion Settings" ?

### DISCUSSION THREAD

**Discussion by bryanseger** [sn4k3/UVtools#855] [Ideas]
My colleagues and I often change the "wait time before cure" and "wait time after cure" settings per resin and/or per machine that we're using. We're often changing this several times a day. We have post-it notes on our desks where we have these settings defined per resin/machine, but it would be very helpful if we could save these settings per profile (resin and/or machine) within UVtools. Having a dropdown list within this screen where we can quickly pull these values in from a previous save would make our time more efficient and have less risk in having an error. Would this be easy to implement in a future update?

![image](https://github.com/sn4k3/UVtools/assets/4185392/edc0f286-54e7-488b-a6c4-7db88a973f13)

**Comment by sn4k3** [comment]
It will defenitly take some effort to implement. But atm it support import / export the settings. Instead of post-it you can use files.
